src/phoenix/server/api/types/DatasetVersion.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+from strawberry.relay import Node, NodeID
+from strawberry.scalars import JSON
+
+
+@strawberry.type
+class DatasetVersion(Node):
+    id_attr: NodeID[int]
+    description: Optional[str]
```

Comment:
```suggestion
    description: Optional[str] = None
```

Not sure if this is the desired behavior, but this is currently a required argument.

Reply:
It's correct in the generated schema. It's not a param so I think it's fine?
```
+type DatasetVersion implements Node {
+  """The Globally Unique ID of this object"""
+  id: GlobalID!
+  description: String
+  metadata: JSON!
+  createdAt: DateTime!
+}
+
+"""A connection to a list of items."""
+type DatasetVersionConnection {
+  """Pagination data for this connection"""
+  pageInfo: PageInfo!
+
+  """Contains the nodes in this connection"""
+  edges: [DatasetVersionEdge!]!
+}
+
+"""An edge in a connection."""
+type DatasetVersionEdge {
+  """A cursor for use in pagination"""
+  cursor: String!
+
+  """The item at the end of the edge"""
+  node: DatasetVersion!
+}
+
```